component/conprof/scrape/scrape.go

```diff
@@ -79,7 +78,7 @@ func (sl *ScrapeSuite) run(ticker *TickerChan) {
 		if scrapeErr == nil {
 			if buf.Len() > 0 {
 				sl.lastScrapeSize = buf.Len()
-				ts := util.GetTimeStamp(start)
+				ts := start.Unix()
 				err := sl.store.AddProfile(meta.ProfileTarget{
```

Comment:
If your `AddProfile` accepts `time.Time` and internally convert a time to ts (since it cannot store time directly) then you reduced a place of "what's the precision of ts" question.

Reply:
great, done.